integration/docker/entrypoint.sh

```diff
@@ -115,6 +116,10 @@ function mountAlluxioFSWithFuseOption {
   fi
 }
 
+function mountFuseSDK {
+  exec bin/alluxio-fuse mount "${@}" -f
```

Comment:
Since the WOKDIR is `/opt/alluxio` in Dockerfile, I suppose this command should be `integration/fuse/bin/alluxio-fuse mount ...`, just like the normal fuse?

Reply:
the fuse sdk tarball is
bin/alluxio-fuse
lib/fuse-jar & ufs jars
conf/
logs/
libexec/

So no need to use integration/fuse/bin/